### DISCUSSION TITLE
Is the project still being maintained?

### DISCUSSION THREAD

**Discussion by jonkerj** [netbox-community/devicetype-library#2381] [General]
We recently took some time to improve this repo by fixing some small things (#2364, #2369) and add a lot of devices (#2368), because we believe contributing and sharing is key to open source. After having done so, I noticed this project has a lot of open PRs without any comment to them, which worries me.

Is this project still being maintained?

**Comment by danner26** [comment]
Hello, yes it is. I have not had any spare cycles to work on this though and I am the only one that focuses on this at the moment. The other maintainers are very busy maintaining and building the main application.
